src/store/traction/mutations.js

```diff
@@ -1,9 +1,6 @@
 import defaultState from './state'
 
 const mutations = {
-  setTags(state, tags) {
-    state.tractionTags = tags
-  },
```

Comment:
Do we no longer require setTags?

Reply:
All tag related stuff has moved to v2. This is all in Pinia store. This is no longer required.